public/index.html

```diff
@@ -8,15 +8,22 @@
     <title>
         <%= title %>
     </title>
-    <link rel="icon" href="favicon.ico">
+    <link rel="icon" href="./favicon.ico">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
         integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+    <script>
+        // note: defaults, the template engine is `Eta`, to pass unescaped vars, use the `~` char instead of `=`
+        var locales = <%~ locales %>;
+    </script>
+    <script src="../src/index.js" defer="defer"></script>
```

Comment:
specify source JS and SCSS/CSS files directly in HTML with path relative to HTML file